clients/admin-ui/src/pages/data-catalog/[systemId]/projects/[projectId].tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  getCoreRowModel,
+  getExpandedRowModel,
+  getGroupedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { useRouter } from "next/router";
+import { useEffect, useMemo } from "react";
+
+import Layout from "~/features/common/Layout";
+import { DATA_CATALOG_ROUTE } from "~/features/common/nav/v2/routes";
+import PageHeader from "~/features/common/PageHeader";
+import {
+  FidesTableV2,
+  PaginationBar,
+  TableSkeletonLoader,
+  useServerSidePagination,
+} from "~/features/common/table/v2";
+import EmptyCatalogTableNotice from "~/features/data-catalog/datasets/EmptyCatalogTableNotice";
+import useCatalogDatasetColumns from "~/features/data-catalog/datasets/useCatalogDatasetColumns";
+import { useGetMonitorResultsQuery } from "~/features/data-discovery-and-detection/discovery-detection.slice";
+import { useGetSystemByFidesKeyQuery } from "~/features/system";
+import { StagedResourceAPIResponse } from "~/types/api";
+
+const EMPTY_RESPONSE = {
+  items: [],
+  total: 0,
+  page: 1,
+  size: 50,
+  pages: 1,
+};
+
+// dataset view when projects are present
+
+const CatalogDatasetView = () => {
+  const { query, push } = useRouter();
+  const systemKey = query.systemId as string;
+  const projectId = query.projectId as string;
+
+  const { data: system, isLoading: systemIsLoading } =
+    useGetSystemByFidesKeyQuery(systemKey);
+
+  const {
+    PAGE_SIZES,
+    pageSize,
+    setPageSize,
+    onPreviousPageClick,
+    isPreviousPageDisabled,
+    onNextPageClick,
+    isNextPageDisabled,
+    startRange,
+    endRange,
+    pageIndex,
+    setTotalPages,
+  } = useServerSidePagination();
+
+  const {
+    isFetching,
+    isLoading,
+    data: resources,
+  } = useGetMonitorResultsQuery({
+    staged_resource_urn: projectId,
+    page: pageIndex,
+    size: pageSize,
+  });
+
+  const {
+    items: data,
+    total: totalRows,
+    pages: totalPages,
+  } = useMemo(() => resources ?? EMPTY_RESPONSE, [resources]);
+
+  useEffect(() => {
+    setTotalPages(totalPages);
+  }, [totalPages, setTotalPages]);
+
+  const columns = useCatalogDatasetColumns();
+
+  const tableInstance = useReactTable<StagedResourceAPIResponse>({
+    getCoreRowModel: getCoreRowModel(),
+    getGroupedRowModel: getGroupedRowModel(),
+    getExpandedRowModel: getExpandedRowModel(),
+    columns,
+    manualPagination: true,
+    data,
+    columnResizeMode: "onChange",
+  });
+  const showContent = !isLoading && !systemIsLoading && !isFetching;
+
+  if (isLoading || isFetching || systemIsLoading) {
+    return <TableSkeletonLoader rowHeight={36} numRows={36} />;
+  }
```

Comment:
Since this falls outside of the `Layout` the skeleton is taking up the entire page (bleeding edge to edge) which I don't think is the desired effect. This should probably be moved in lower.